README.adoc

```diff
@@ -3,6 +3,7 @@
 image:https://travis-ci.org/jaegertracing/jaeger-operator.svg?branch=master["Build Status", link="https://travis-ci.org/jaegertracing/jaeger-operator"]
 image:https://goreportcard.com/badge/github.com/jaegertracing/jaeger-operator["Go Report Card", link="https://goreportcard.com/report/github.com/jaegertracing/jaeger-operator"]
 image:https://codecov.io/gh/jaegertracing/jaeger-operator/branch/master/graph/badge.svg["Code Coverage", link="https://codecov.io/gh/jaegertracing/jaeger-operator"]
+image:https://godoc.org/github.com/jaegertracing/jaeger-operator?status.svg["GoDoc", link="https://godoc.org/github.com/jaegertracing/jaeger-operator/pkg/apis/jaegertracing/v1#JaegerSpec"]
```

Comment:
The link goes directly to the `JaegerSpec` as it is the only exposed API. I want people to start using godoc as our temporary CR documentation

Reply:
+1 to add to add the link to godoc, but I'd not direct people to it for CR documentation. Godoc might be too confusing for people to grasp. I'd rather point them to our examples directory and, as last resort, tell that they could check directly the code (and then link Godoc)

Reply:
The godoc itself is useless nobody will import jaeger-operator.

My only objective here is to provide a quick link to full CR struct. Our examples show only a fraction of what is exposed in the CR.